src/lightning_app/utilities/app_logs.py

```diff
@@ -66,8 +77,13 @@ def _error_callback(ws_app: WebSocketApp, error: Exception):
 
 
 def _app_logs_reader(
-    client: LightningClient, project_id: str, app_id: str, component_names: List[str], follow: bool
-) -> Iterator[Tuple[str, _LogEvent]]:
+    client: LightningClient,
+    project_id: str,
+    app_id: str,
+    component_names: List[str],
+    follow: bool,
+    on_error_callback: Optional[Callable] = None,
+) -> Iterator[_LogEvent]:
 
     read_queue = queue.PriorityQueue()
```

Comment:
Confirm we are sorting queue items using `ge` `le` relationships between events right?

Reply:
No, it isn't properly sorted yet. We would need a custom queue to resolve this.

Reply:
I see, then I'm wondering why we are using a `PriorityQueue`